tests/unit/frameworks/models/motor_policies_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+import unittest
+from typing import Dict
+
+import numpy as np
+
+from tbp.monty.frameworks.actions.action_samplers import UniformlyDistributedSampler
+from tbp.monty.frameworks.actions.actions import LookUp
+from tbp.monty.frameworks.models.motor_policies import BasePolicy
+from tbp.monty.frameworks.models.motor_system_state import (
+    AgentState,
+    MotorSystemState,
+    SensorState,
+)
+
+
+class BasePolicyTest(unittest.TestCase):
+    def setUp(self):
+        self.rng = np.random.RandomState(42)
+        self.agent_id = f"agent_id_{self.rng.randint(0, 999_999_999)}"
+        self.default_sensor_state: SensorState = {
+            "position": (0.0, 0.0, 0.0),
+            "rotation": (1.0, 0.0, 0.0, 0.0),
+        }
+        self.agent_sensors: Dict[str, SensorState] = {
+            f"sensor_id_{self.rng.randint(0, 999_999_999)}": self.default_sensor_state,
+        }
```

Comment:
Do the old ways of defining agent state still work (just nested dictionaries)? Do I need to update anything in the tutorials (open PR)? I think it's fine because the conversion happens after the state dict is returned from the dataloader, right?

Reply:
I was thinking of the `get_state` function in the DataLoader (like here https://github.com/thousandbrainsproject/tbp.monty/blob/6bf52739e17fa68cf047556618b9507408016725/src/tbp/monty/frameworks/environments/two_d_data.py#L149 ) But looking at it now I see that your PR won't change anything about that, it just converts once it gets to the motor system.